tcks/profiles/platform/docs/userguide/platform/src/main/asciidoc/config.adoc

```diff
@@ -64,6 +64,30 @@ See <<configuring-your-application-server-as-the-vi>> for a list of the modifica
 configuration is valid. See <<[validating-your-test-configuration>> for information about using Maven/Arquillian to run the sample tests.
 6.  Run the TCK tests. See <<executing-tests>> for information
 about using Maven to start running tests.
+7. Generate
+
+## Generate a Client Certificate
+
+Some tests require a client certificate to be installed on the Jakarta EE {tck_version} server. The following steps describe how to generate a client certificate:
+[source,bash]
+----
+keytool -genkeypair -alias cts -keyalg RSA -keysize 2048 -validity 365 -keystore clientcert.p12 -dname "CN=CTS, OU=Eclipse Foundation, O=Jakarta EE" -keypass changeit -storepass changeit
+
+Generating 2,048 bit RSA key pair and self-signed certificate (SHA256withRSA) with a validity of 365 days
+for: CN=CTS, OU=Eclipse Foundation, O=Jakarta EE
+
+keytool -exportcert  -keystore clientcert.p12 -alias cts -keypass changeit -storepass changeit -file cts_cert.cer
```

Comment:
Just thinking out loud a bit here. Should we mention something about how this is used? TBH I'm not quite sure, but IIRC you have to do something with server trust store or server cert. I need to brush up on this lol

Reply:
You will have to use the server's docs for importing the client cert into its truststore.

Reply:
Got it and makes sense.